### ISSUE TITLE
undefined is not a function when passing mime type to the useDropzone hook

### ISSUE THREAD

**Issue by shoaib-vaival** [react-dropzone/react-dropzone#1337]
**Describe the bug**

A build error is occurring in Next.js with the message "undefined is not a function." This issue arises when attempting to pass a MIME type to the useDropzone hook while using the Next.js app router.

**To Reproduce**

1. Create a Generic FileUpload component 
2. import it in react server component and provide mime type according to document 
3. run npm or yarn build

**Comment by userAdityaa** [comment]
Hello, I'm interested in this issue. But, im new to github. so, im not able to recreate this issue can you help me with it.

**Comment by shoaib-vaival** [comment]
@userAdityaa yes sure, you can create this just using React dropzone in server side component.

**Comment by userAdityaa** [comment]
Im sorry but im still not able to reacreate this can you help me with this.

**Comment by rolandjitsu** [comment]
@shoaib-vaival please provide a codesandbox that illustrates the issue.
